cloud/terraform/templates.go

```diff
@@ -213,7 +213,11 @@ sysctl net.bridge.bridge-nf-call-iptables=1
 # kubeadm uses 10th IP as DNS server
 CLUSTER_DNS_SERVER=$(prips ${SERVICE_CIDR} | head -n 11 | tail -n 1)
 
-sed -i "s/KUBELET_DNS_ARGS=[^\"]*/KUBELET_DNS_ARGS=--cluster-dns=${CLUSTER_DNS_SERVER} --cluster-domain=${CLUSTER_DNS_DOMAIN}/" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+cat > /etc/systemd/system/kubelet.service.d/20-cloud.conf << EOF
+[Service]
```

Comment:
Why is this needed? (it seems difficult to track down on google what exactly this block enables)

Reply:
These are part of the systemd configuration files for kubelet that are installed as part of the kubeadm package. We were sed editing an existing one, but it's cleaner to write a new file with a higher number to 
override things.

These are ultimately kubelet flags. The gce startup scripts switched to this approach as well. https://github.com/kubernetes-sigs/cluster-api/blob/master/gcp-deployer/machine_setup_configs.yaml#L74-L79